### ISSUE TITLE
Tag mode opens file in the nerdtree window

### ISSUE THREAD

**Issue by ChaoMai** [ctrlpvim/ctrlp.vim#136]
In my .vimrc, I just set the `g:ctrlp_working_path_mode`, `g:ctrlp_extensions` and some key mappings about ctrlp.

If I have nerdtree opened. When I use tag mode, the newly opened file is in the nerdtree window.

For example, I have 3 files and a tags generated by ctag.

![1](https://cloud.githubusercontent.com/assets/1412813/10118725/748ad91a-64b2-11e5-96e1-a7d5747256b9.JPG)

Util now, every thing is ok.

![2](https://cloud.githubusercontent.com/assets/1412813/10118727/7858de16-64b2-11e5-8728-e2c528bb9fdb.JPG)

But after I hit enter, the file a.c is opened in the nerdtree window.

I've tried some options like `g:ctrlp_reuse_window`. But it doesn't seem works.

Would you please tell me how to adjust the configuration of the ctrlp, so that I can open the a.c in the right window?

Thank you

**Comment by pawel-slowik** [comment]
When looking for a buffer to open the selected file in, ctrlp skips `quickfix` and `help` buffers:
```
fu! ctrlp#normcmd(cmd, ...)
        let buftypes = [ 'quickfix', 'help' ]
        if a:0 < 2 && s:nosplit() | retu a:cmd | en
        let norwins = filter(range(1, winnr('$')),
                \ 'index(buftypes, getbufvar(winbufnr(v:val), "&bt")) == -1 || s:isneovimterminal(winbufnr(v:val))')
```
(in autoload/ctrlp.vim).

The nerdtree buffer's type is `nofile`, therefore it is **not** skipped and may end up selected by ctrlp.

Changing the line to:
```
let buftypes = [ 'quickfix', 'help', 'nofile' ]
```
fixes this issue for me.

**Comment by qsvui** [comment]
Hello, I was wondering when or if this fix will be included.

**Comment by mattn** [comment]
closed by #425
